Packs/MandiantAdvantageAttackSurfaceManagement/Integrations/AttackSurfaceManagement/AttackSurfaceManagement.py

```diff
@@ -0,0 +1,520 @@
+
+        message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):  # TODO: make sure you capture authentication errors
+            message = 'Authorization Error: make sure Access Key and Secret Key is correctly set'
+        else:
+            raise e
+    return message
+
+
+def get_projects(client: Client, args: dict = None) -> CommandResults:
+    args = args if args else {}
+
+    response = client.get_projects_list()
+
+    output = [{'Name': proj['name'],
+               'ID': proj['id'],
+               'Owner': proj['owner_email']} for proj in response]
+
+    return CommandResults(
+        outputs_prefix='MandiantAdvantageASM.Projects',
+        outputs=output,
+        ignore_auto_extract=True
+    )
+
+
+def get_collections(client: Client, args: dict = None) -> CommandResults:
+    args = args if args else {}
+
+    project_id: int = args.get('project_id', client.project_id)
+
+    response = client.get_collections_list(project_id)
+
+    output = [{'Name': collection['printable_name'],
+               'ID': collection['name'],
+               'Owner': collection['owner_name']} for collection in response]
+
+    return CommandResults(
+        outputs_prefix='MandiantAdvantageASM.Collections',
+        outputs=output,
+        ignore_auto_extract=True
+    )
+
+
+
+
+def fetch_incidents(client: Client):
+    last_run = demisto.getLastRun()
+    last_start_time: datetime
+
+    params = demisto.params()
+
+    if not client.project_id:
+        raise DemistoException('Must configure a Project ID to fetch incidents from Mandiant ASM')
+
+    if last_run and 'start_time' in last_run and last_run['start_time']:
+        last_start_time = parser.parse(last_run.get('start_time'))
+    else:
+        parsed_time = dateparser.parse(params.get("first_fetch"), settings={'TIMEZONE': 'UTC', 'RETURN_AS_TIMEZONE_AWARE': True})
+        if not parsed_time:
+            parsed_time = datetime.now()
+        last_start_time = parsed_time
+
+    issues = client.get_issues_since_time(last_start_time)
+    most_recent_update = None
+
+    parsed_issues = []
+
+    for issue in issues:
+        incident_details = client.get_issue_details(issue["id"])
+        new_incident = helper_create_incident(incident_details, client.project_id)
+        new_incident['dbotMirrorLastSync'] = datetime.fromtimestamp(0).strftime("%Y-%m-%dT%H:%M:%SZ")
+        new_incident['lastupdatetime'] = new_incident['dbotMirrorLastSync']
+
+        parsed_issues.append(new_incident)
+        most_recent_update = issue['last_seen']
+
+        if len(parsed_issues) >= client.limit:
+            break
+
+    parsed_issues = filter_incidents_by_duplicates_and_limit(
+        incidents_res=parsed_issues, last_run=last_run, fetch_limit=client.limit, id_field='dbotMirrorId'
+    )
+
+    new_issues = [json.loads(i['rawJSON'])['uid'] for i in parsed_issues]
+
+    last_run = {
+        'start_time': most_recent_update,
```

Comment:
```suggestion
        'time': most_recent_update,
```

i would call it just `time`